client/app/scripts/stores/app-store.js

```diff
@@ -357,7 +372,13 @@ export class AppStore extends Store {
       this.__emitChange();
       break;
 
+    case ActionTypes.CLICK_PAUSE_UPDATE:
+      updatePausedAt = new Date;
```

Comment:
I didn't realise parens were optional in this case! TIL. Though would still go for. `new Date()`